src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
+    - include:
+          file: /db/changelog/db.changeset-init-schema.sql
+    - includeAll:
+          path: db/changelog/2024/09
```

Comment:
there are multiple ways to design liquibase projects, primarily two - either `object` oriented or `release` oriented, you can read about those here - https://docs.liquibase.com/start/design-liquibase-project.html